src/ControllerMessenger.ts

```diff
@@ -300,10 +301,10 @@ export class RestrictedControllerMessenger<
    *
    * The event type being unsubscribed to must be on the event allowlist.
    *
-   * @param eventType - The event type. This is a unique identifier for this event.
+   * @param event - The event type. This is a unique identifier for this event.
    * @param handler - The event handler to unregister.
    * @throws Will throw when the given event handler is not registered for this event.
-   * @template T - A type union of allowed Event type strings.
+   * @template T- A type union of allowed Event type strings.
```

Comment:
Why is the space getting removed here 🤔

Reply:
Great catch! I think one of the `yarn lint:fix` runs must have messed this up while it was trying to fix something else. These have been fixed.

Reply:
I've just updated the config to ensure this doesn't happen again in https://github.com/MetaMask/controllers/pull/591/commits/1bb40df3dd7a9e445da3c3f4beb145bcb6da8e4f